.devcontainer/devcontainer.json

```diff
@@ -32,11 +32,22 @@
 
 	// Add the locally installed dotnet to the path to ensure that it is activated
 	// This allows developers to just use 'dotnet build' on the command-line, and the local dotnet version will be used.
+	// Add the global tools dir to the PATH so that globally installed tools will work
 	"remoteEnv": {
-		"PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}",
-		"DOTNET_MULTILEVEL_LOOKUP": "0"
+		"PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}:${containerWorkspaceFolder}/.dotnet-tools-global",
```

Comment:
Why did you put this at the end rather than after the `.dotnet` folder?

Reply:
Will fix

Reply:
Fixed